tools/slang-reflection-test/slang-reflection-test-tool.vcxproj.filters

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I have no idea what the diff is here. Is it something where you can just revert the file and get rid of it?

Reply:
reverted.